### DISCUSSION TITLE
Multiple Files On Flash Memory

### DISCUSSION THREAD

**Discussion by chuck808** [pschatzmann/arduino-audio-tools#166] [Q&A] [answered]
Hi,

Trying to find a way to call small WAV files from flash memory, something like;

play->sample1.wav
play->sample2.wav

Is this possible?

Thanks

**Comment by pschatzmann** [comment, marked as answer]
Sure it should not be too difficult to have this implemented using the audio-tools. I suggest to use [this example](https://github.com/pschatzmann/arduino-audio-tools/tree/main/examples/examples-stream/streams-memory_wav-pwm) as starting point.

Or you could use [SPIFFS](https://github.com/pschatzmann/arduino-audio-tools/issues/133) and just handle them as files.

Or you could use my [simple-tts library](https://github.com/pschatzmann/arduino-simple-tts) which actually does something similar based on the audio-tools: You just collect all WAV files into a dictionary and play them with the say command.

Since the wav files are not compressed, I recommend to use mono and a low sampling rate to keep the files as small as possible.
